edubots_surveys.Rnw

```diff
@@ -713,31 +691,14 @@ analysing the response to the different survey questions.
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 
-Group messaging apps have become a powerful tool for \textcolor{blue}{communicating} with
```

Comment:
Not clear what you have done here. I would have very much preferred to see very atomic changes. It's not clear now to me if this has been deleted, or simply moved elsewhere.